src/content/developers/docs/spin-up-your-node/index.md

```diff
@@ -0,0 +1,146 @@
+Client execution will initiate its core functions, chosen endpoints and start looking for peers. After successfully discovering peers, client starts synchronization. Current blockchain data will be available once client is successfully synced to the current state. 
+
+### Using the client {#using-the-client}
+
+Client software is running in the background. For interacting with it, implementations offer RPC API endpoints. Controlling the client and interacting with Ethereum network can be done in various ways using these RPC endpoints:
+- Manualy by calling them with suitable protocol
+- Attaching provided console 
+- By applications implementing them
+
+Implementations of RPCs endpoints and methods can slightly differ between clients. Generally spread is JSON-RPC which can be used with all client implementations. For general overview, refer to [this eth.wiki article. ](https://eth.wiki/json-rpc/API). 
+This RPC can be used by applications requiring info from Ethereum network, for example wallets. For instance, popular wallet Metamask [allows easily add RPC URL to use local instance](https://metamask.zendesk.com/hc/en-us/articles/360015290012-Using-a-Local-Node). 
+
+#### Reaching RPC {#reaching-rpc}
+Ports of all local endpoints can be modified in config, default port of JSON-RPC is `8545`. This RPC interface is by default reachable only on localhost of the computer. If you want to directly expose it to public,its address can be changed to `"0.0.0.0"`. This way, it will be reachable over local and public IP address. In most cases, to reach endpoint publicly you will also need to setup port forwarding on the router. 
+
+But beware, because this way anybody on the internet can control your node. Malicious actors could misuse it to e.g. halt your system or steal funds if client is used as wallet. Which RPC methods are allowed can be modified in configuration, those potentially harmful can be excluded. For example while starting `geth`, you can determine allowed method sets with a flag `--http.api web3,eth,txpool`. 
+
+Other ways how to expose RPC can be indirectly using web server like Nginx. You can easily host access to the RPC by pointing web service to client's local address and port. 
+
+For privacy preserving and easy to setup publicly reachable endpoint, hosting it on own [Tor](https://www.torproject.org/) onion service might be a good option. This enables you to reach RPC outside your local network even without static public IP address and opened ports. 
+After installing `tor`, edit `torrc` config and configure hidden service pointing to client's RPC address and port. After restarting tor, it will generate hidden service keys and hostname in desired directory. Your RPC will be reachable on this `.onion` hostname.
```

Comment:
To do this:
1. Install `tor`
2. Edit `torrc` config 
3. _not sure how to word the third point exactly... Configure the hidden service to point to your client's RPC address and port?_

Once you restart Tor, you'll get hidden service keys and a hostname in your desired directory. From then, your RPC will be reachable on a `.onion` hostname.

Reply:
The most privacy-preserving and simple way to set up a publicly reachable endpoint, you can host it on your own [Tor](https://www.torproject.org/) onion service. This will let you reach the RPC outside your local network without a static public IP address or opened ports. 
To do this:
- Install `tor`
- Edit `torrc` config to enable hidden service with address of your client's RPC address and port
- Restart `tor` service